packages/app/src/app/components/SubscribeForm/index.tsx

```diff
@@ -2,33 +2,39 @@ import React from 'react';
 import { StripeProvider, Elements } from 'react-stripe-elements';
 import { STRIPE_API_KEY } from '@codesandbox/common/lib/utils/config';
 import CheckoutForm from './CheckoutForm';
-
 import { Container } from './elements';
 
-function SubscribeForm({
+interface ISubscribeFormProps {
```

Comment:
@Saeris I think you should use a normal `type` instead of an `interface`

```suggestion
type SubscribeFormProps = {
```